portalnet/src/overlay.rs

```diff
@@ -540,6 +545,20 @@ where
             );
         }
     }
+
+    pub fn get_summary_info(&self) -> String {
+        format!(
+            "offerin={}, acceptout={}, offerout={}, acceptin={}",
```

Comment:
IMO the extra character is worth the legibility

```suggestion
            "offer-in={}, accept-out={}, offer-out={}, accept-in={}",
```

Reply:
Hah, I was just thinking about how to make it even shorter, since we'll end up with even more stuff in there. I landed on:
```
023-04-13T01:45:10.714353Z  INFO trin_history: storage-report: radius=0.0% content=2086.4/2000kb #=30 disk=5407.0kb; msg-report: offered=523/526, accepted=656/656
```

Reply:
Yup, exactly! I feel like it's a good sign that you intuited the explanation.